packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -1941,7 +1944,15 @@ describe('TransactionController', () => {
   });
 
   describe('stopTransaction', () => {
-    it('rejects result promise', async () => {
+    it('submits a cancel transaction', async () => {
```

Comment:
Is it worth splitting this for clarity so we assert in one test that it submits, and in another that it persists a new metadata entry?

Reply:
Tests are separated in https://github.com/MetaMask/core/pull/1998/commits/be146612e7d24db8999065eeccee29ee6701f2d4